File: flutter_cab_pwa/lib/models/login_request.dart
// models/login_request.dart

class LoginRequest {
  final String? webLogin;
  final String? password;

  LoginRequest(this.webLogin, this.password);

  Map<String, dynamic> toJson() {
    return {
      "web_login": webLogin,
      "password": password
    };
  }
}

File: CabPwaApi/CabPwaApi/Requests/LoginRequest.cs
// Requests/LoginRequest.cs
using System.Text.Json.Serialization;

namespace CabPwaApi.Requests
{
    public class LoginRequest
    {
        [JsonPropertyName("web_login")]
        public string web_login { get; set; } = string.Empty;

        [JsonPropertyName("password")]
        public string password { get; set; } = string.Empty;
    }
}
